lib/optimize.js

```diff
@@ -76,6 +76,12 @@ var optimizer = {
             limit = Math.min(limit, optimization_info.limit);
         }
 
+        var read_fetch_size = graph.node_get_option(read, 'fetch_size');
```

Comment:
what if `fetch_size` is not indicated in the juttle? Consider using the default value?